tools/vpn/vpn_softetherbridge.md

```diff
@@ -176,32 +202,34 @@ You may adjust DHCP setting for physical interfaces.
 Further information about local bridge is [here](https://www.softether.org/4-docs/1-manual/3._SoftEther_VPN_Server_Manual/3.6_Local_Bridges#3.6.11_Points_to_Note_when_Local_Bridging_in_Linux.2C_FreeBSD.2C_Solaris_or_Mac_OS_X).
 
 ## 9. Automatically starting at boot
+
 The local VPN bridge software automatically saves its configuration in a file (`vpn_bridge.config`, owned by root) so
 you do not have to reconfigure it each time you run it.
-If you set it up to run in 
-["service mode"](https://www.softether.org/4-docs/1-manual/7._Installing_SoftEther_VPN_Server/7.3_Install_on_Linux_and_Initial_Configurations) 
+If you set it up to run in
+["service mode"](https://www.softether.org/4-docs/1-manual/7._Installing_SoftEther_VPN_Server/7.3_Install_on_Linux_and_Initial_Configurations)
 then it will also run automatically at boot using `init.d`.
 This starts by copying the files to `/usr/local` and making them owned by root,
 then setting up an `init.d` service file.  However the instructions for the service file are not
 quite right for the Raspberry Pi which does not have `chkconfig`.
-Instead you have to 
-manually set the dependencies in the service file (see here for an appropriate 
-[init.d/vpnbridge service file](vpnbridge)) and then use the following to 
+Instead you have to
+manually set the dependencies in the service file (see here for an appropriate
+[init.d/vpnbridge service file](vpnbridge)) and then use the following to
```

Comment:
```suggestion
[`init.d/vpnbridge` service file](vpnbridge)) and then use the following to
```

Reply:
Why not present the CLI data as CLI?

(Why mark a comment as resolved without applying it nor commenting to explain why *not* applying it? This may result in the commenter accepting your reasoning, or presenting counter-argument, which you might then accept or present further argument against....)